components/Poll/Poll.jsx

```diff
@@ -0,0 +1,342 @@
+        usersVoted: [],
+      });
+      this.props.savePoll({
+        title: this.state.title,
+        multiSelect: this.state.multiSelect,
+        open: this.state.open,
+        options: opts,
+      });
+    }
+  }
+
+  buildListItems = () => {
+    let items = this.state.options
+            .sort((a, b) => b.usersVoted.length - a.usersVoted.length)
+            .slice()
+            .map(item => {
+              if (item._id) {
+                item.key = item._id;
+              }
+              return item;
+            });
+    if (!!this.props.isAuthor || this.state.open) {
+      items.push({ key: 'add_option_item' });
+    }
+    return items;
+  }
+
+  renderListItem = ({ item }) => {
+    if (item.key === 'add_option_item') {
+      return (
+        <View style={styles.optionGroup}>
+          <TouchableOpacity style={styles.addContainingView} onPress={this.addOption}>
+            <Icon name={Platform.OS === 'ios' ? "ios-add" : "md-add"} style={styles.addView}/>
+          </TouchableOpacity>
+          <Textarea
+            placeholder="Add an option..."
+            style={styles.addOptionText}
+            onChangeText={this.newOptionUpdate}
+            value={this.state.newOption}
+          />
+        </View>
+      );
+    }
+    let selected = (this.userVoteIndex(item) !== -1);
+    return(
+  		<TouchableOpacity style={styles.fillWidth} onPress={this.toggleOption.bind(this, item)} onLongPress={this.confirmDelete.bind(this, item)}>
+        <View style={styles.optionView}>
+          {this.state.multiSelect ?
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            /> :
+            <CheckBox
+              style={styles.optionButton}
+              containerStyle={styles.optionButtonContainer}
+              checkedIcon='dot-circle-o'
+              uncheckedIcon='circle-o'
+              checked={selected}
+              onIconPress={this.toggleOption.bind(this, item)}
+              onLongIconPress={this.confirmDelete.bind(this, item)}
+            />}
+  			  <Text style={styles.optionText}>{item.text}</Text>
+  			  <Text style={styles.optionCount}>{`${item.usersVoted.length}`}</Text>
+        </View>
+  		</TouchableOpacity>
+    );
+  }
+
+  render() {
+    // TODO fix: so many security vulnerabilities from unsanitized user input of option text and poll title
+    let editing = !!this.props.savePoll;
+
+    return (
+      <View style={styles.view}>
+        {!!this.props.isAuthor && editing ?
+          <View style={styles.fillWidth}>
+            <CheckBox
+              style={styles.topCheckbox}
+              containerStyle={styles.topCheckboxContainer}
+              title={`${this.state.open ? 'Disallow' : 'Allow'} Option Creation`}
```

Comment:
Can we just keep this as Allow and use the same icon as "allow multiple selections"? Keeps it consistent and easier to understand :)

Reply:
Should be fixed in most recent update